cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityWritePrivilegesTest.scala

```diff
@@ -14,23 +14,208 @@ class SecurityWritePrivilegesTest extends DocumentingTest with QueryStatisticsTe
+      query("GRANT SET PROPERTY {*} ON GRAPH neo4j ELEMENTS * TO regularUsers", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 2)
+      })) {
+        statsOnlyResultTable()
+      }
+
+      p("The `SET PROPERTY` privilege can also be denied.")
+      p("include::deny-set-property-syntax.asciidoc[]")
+
+      p("For example, denying the ability to set the property `foo` on nodes with the label `bar` on all graphs to the role `regularUsers` would be achieved using:")
+      query("DENY SET PROPERTY { foo } ON GRAPH * NODES bar TO regularUsers", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      note {
+        p("If the users attempts to set a property with a property name that does not already exist on the database the user must also possess the " +
+          "<<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>> privilege.")
+      }
+    }
+
+    section("The `MERGE` privilege", "administration-security-writes-merge", "enterprise-edition") {
+      p(
+        """The `MERGE` privilege is a compound privilege that combines `TRAVERSE` and `READ` (i.e. `MATCH`) with `CREATE` and `SET PROPERTY`. This is intended to
+          | permit use of <<query-merge, the MERGE command>> but is applicable to all reads and writes that require these privileges.""".stripMargin)
+      p("include::grant-merge-syntax.asciidoc[]")
+
+      p(
+        """For example, granting `MERGE` on all elements of the graph `neo4j` to the role `regularUsers` would be achieved using:""".stripMargin)
+      query("GRANT MERGE {*} ON GRAPH neo4j ELEMENTS * TO regularUsers", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 2)
+      })) {
+        statsOnlyResultTable()
+      }
+
+      p("It is not possible to deny the `MERGE` privilege. If it is desirable to prevent a users from creating elements and setting properties use " +
+        "<<administration-security-writes-create, DENY CREATE>> or <<administration-security-writes-set-property,DENY SET PROPERTY>>.")
+
+      note {
+        p("If the users attempts to create nodes with a label that does not already exist on the database the user must also possess the " +
+          "<<administration-security-administration-database-tokens, CREATE NEW LABEL>> privilege. The same applies to new relationships and properties - the " +
+          "<<administration-security-administration-database-tokens, CREATE NEW RELATIONSHIP TYPE>> or " +
+          "<<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>> privileges are required.")
+      }
+    }
+
     section("The `WRITE` privilege", "administration-security-writes-write", "enterprise-edition") {
       p(
-        """The `WRITE` privilege enables you to write on a graph.""".stripMargin)
+        """The `WRITE` privilege allows the user to execute any write command on a graph.""".stripMargin)
```

Comment:
```suggestion
        """The `WRITE` privilege enables the user to execute any write command on a graph.""".stripMargin)
```

Reply:
I see this was previously "enables". 

In the context of privileges, the difference between enabling and allowing is thin. I think we should default to consistency.

Reply:
I think we have an opportunity to highlight a key difference between READ and WRITE privileges. See my comment above. In this particular case we should use _allows_ because we would result in an error without it.